frontend/src/console/components/AlertMessage/AlertMessage.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Alert } from 'react-bootstrap';
+import './styles.scss';
+import { WrappedMessage } from 'utils/intl';
+import messages from './displayMessages';
+
+export const AlertMessage: React.FC = () => (
+  <Alert
+    className="d-flex justify-content-center align-middle alert-warning"
+    variant="warning"
+  >
+    <WrappedMessage id="verifyEmail" messages={messages} />
+  </Alert>
+);
```

Comment:
Since this component has a generic name like `AlertMessage` it shouldn't be fixed to just showing an alert message about email. Either the component should be called `EmailVerificationAlert` or it should be generic and allow any alert message. 

Can you move this `WrappedMessage` to a parameter to this component?
i.e. 

```suggestion

interface Props {
  children: React.ReactNode;
}

export const AlertMessage: React.FC = (props: Props) => (
  <Alert
    className="d-flex justify-content-center align-middle alert-warning"
    variant="warning"
  >
    {props.children}
  </Alert>
);
```